ddtrace/tracer/option.go

```diff
@@ -253,6 +238,21 @@ func newConfig(opts ...StartOption) *config {
 	for _, fn := range opts {
 		fn(c)
 	}
+	if c.agentURL == nil {
+		c.agentURL = resolveAgentAddr()
+		if url := internal.AgentURLFromEnv(); url != nil {
+			c.agentURL = url
+		}
+	}
+	if c.agentURL.Scheme == "unix" {
+		c.httpClient = udsClient(c.agentURL.Path)
+		c.agentURL = &url.URL{
+			Scheme: "http",
```

Comment:
If the agent URL scheme is `unix` should this one be `unix` here as well?

Reply:
No. This is where we intercept `unix://` URI's and "convert" them into `http://` URIs that the http client can use.